components/log-viewer-webui/client/src/components/Layout/MainLayout.tsx

```diff
@@ -15,45 +15,57 @@ import {
 } from "antd";
 
 import styles from "./MainLayout.module.css";
+import ThemeToggleMenuItem from "./ThemeToggleMenuItem";
 
 
 const {Sider} = Layout;
 
 type MenuItem = Required<MenuProps>["items"][number];
 
-const SIDEBAR_MENU_ITEMS: MenuItem[] = [
+const sidebarTopMenuItems: MenuItem[] = [
+    {type: "divider"},
     {label: <Link to={"/ingest"}>Ingest</Link>, key: "/ingest", icon: <UploadOutlined/>},
     {label: <Link to={"/search"}>Search</Link>, key: "/search", icon: <SearchOutlined/>},
 ];
 
+const sidebarBottomMenuItems = [
+    <ThemeToggleMenuItem key={"theme-toggle"}/>,
+];
+
 /**
  * The main layout of web ui.
  *
  * @return
  */
 const MainLayout = () => {
-    const [collapsed, setCollapsed] = useState(false);
+    const {pathname} = useLocation();
 
     return (
         <Layout className={styles["mainLayout"]}>
             <Sider
-                collapsed={collapsed}
+                collapsedWidth={50}
                 collapsible={true}
                 theme={"light"}
                 width={150}
-                onCollapse={(value) => {
-                    setCollapsed(value);
-                }}
             >
```

Comment:
I guess the point of this change is that we don't actually need to control the collapsed state from this component?